.circleci/config.yml

```diff
@@ -1,19 +1,462 @@
+  working_directory: ~/workspace
+  environment:
+    ROX_CI_IMAGE: true
+
+defaultMachine: &defaultMachine
+  machine:
+    image: << pipeline.parameters.default-machine-image >>
+
+runOnAllTags: &runOnAllTags
+  filters:
+    branches:
+      only:
+      - master
+
+runOnAllTagsWithDockerIOPullCtx: &runOnAllTagsWithDockerIOPullCtx
+  <<: *runOnAllTags
+  context:
+    - quay-rhacs-eng-readonly
+
+runOnAllTagsWithDockerIOPushCtx: &runOnAllTagsWithDockerIOPushCtx
+  <<: *runOnAllTags
+  context:
+    - quay-rhacs-eng-readwrite
+    - quay-rhacs-eng-readonly
+    - quay-stackrox-io-readwrite
+
+orbs:
+  slack: circleci/slack@3.4.2
+  ci-artifacts: stackrox/ci-artifacts-orb@0.1.1
+
+commands:
+  initcommand:
+    steps:
+    - attach_workspace:
+        at: ~/workspace
+    - run:
+        name: Set up environment
+        command: |
+          cat >>"$BASH_ENV" \<<-EOF
+            export WORKSPACE_ROOT="${HOME}/workspace"
+          EOF
+          [[ -f ~/workspace/shared-env ]] && cat ~/workspace/shared-env >>"$BASH_ENV"
+
+  read-pr-metadata:
+    steps:
+    - run:
+        name: Read PR metadata
+        command: |
+          "${SOURCE_ROOT}/.circleci/read-labels.sh" pr-metadata/labels
+
+  gcloud-init:
+    parameters:
+      service-account-env:
+        type: string
+        default: GOOGLE_CREDENTIALS_CIRCLECI_COLLECTOR
+      bucket-permission-check:
+        type: string
+        default: gs://collector-build-cache
+
+    steps:
+    - run:
+        name: Setup GCP env
+        command: |
+          "${CI_ROOT}/gcloud-init/setup-gcp-env.sh" "${<< parameters.service-account-env >>}" << parameters.bucket-permission-check >>
+
+  quay-login:
+    steps:
+    - run:
+        name: Login to Quay with pull credentials
+        command: |
+            docker login -u "$QUAY_RHACS_ENG_RO_USERNAME" -p "$QUAY_RHACS_ENG_RO_PASSWORD" quay.io
+
+  quay-login-push:
+    steps:
+    - run:
+        name: Login to Quay with push credentials
+        command: |
+            docker login -u "$QUAY_RHACS_ENG_RW_USERNAME" -p "$QUAY_RHACS_ENG_RW_PASSWORD" quay.io
+
+  dockerized-pull-cache-image:
+    steps:
+    - run:
+        name: Pull cache image
+        command: |
+          "${CI_ROOT}/dockerized-pull-cache-image/10-pull-cache-image.sh" "${COLLECTOR_DRIVERS_CACHE}" << pipeline.parameters.quay-repo >>
+
+  dockerized-check-drivers-built:
+    steps:
+    - run:
+        name: Check if job should run
+        command: |
+          if [[ ! -s ~/workspace/build-tasks ]]; then
+            echo "Nothing was built, skipping this job"
+
+            "${CI_ROOT}/safe-halt.sh"
+            exit
+          fi
+
+  dockerized-should-run:
+    steps:
+    - run:
+        name: Check if dockerized job should run
+        command: |
+          if [[ ! -f ~/workspace/pr-metadata/labels/run-dockerized-steps ]]; then
+            echo "Skipping dockerized build jobs."
+            "${CI_ROOT}/safe-halt.sh"
+            exit
+          fi
```

Comment:
I don't think you need these dockerized commands, but it shouldn't block merging.